### PR TITLE
Supporting int32 and int64 in Less and Greater

### PR DESCRIPTION
**Pull request by houseroad**
More like a spec fix instead of adding new features. So prefer not bumping the opset version for these two ops (really no need).

### CODE REVIEW

**Comment by spandantiwari** [comment]
Maybe we should consider bumping the opset version for those runtimes that support ONNX 1.2 and 1.3 that did not not have the integer types as part of the implementation of these ops.

**Comment by houseroad** [comment]
@spandantiwari ok, let me update the pr later :-)

File: onnx/defs/logical/defs.cc [hunk 1]
```diff
@@ -86,7 +86,11 @@ ONNX_OPERATOR_SET_SCHEMA(
         .FillUsing(BinaryLogicDocGenerator("greater"))
         .TypeConstraint(
             "T",
-            {"tensor(float16)", "tensor(float)", "tensor(double)"},
+            {"tensor(int32)",
```
Comment (linkerzhang) [inline onnx/defs/logical/defs.cc]: thank you Lu! shall we add all numeric types please?

Comment (houseroad) [reply onnx/defs/logical/defs.cc]: Sure, I will do that and bump up the opset version too

**Comment by houseroad** [comment]
@linkerzhang @spandantiwari comments have been addressed, ping for review :-)

**Pull request merged**